src/common/exec_utils.h

```diff
@@ -49,85 +49,19 @@ namespace common {
           indices are not recorded
  * \return true if any source NDArray need to cast storage
  */
-inline bool SetupDefaultBlobsIn(const std::vector<NDArray>& src,
```

Comment:
What warnings did this cause?

Reply:
this is moving bloating inlines to implementation file as discussed in dev@